README.md

```diff
@@ -1509,6 +1509,7 @@ Update Time, five active automations, webhooks.
   * [Zeplin](https://zeplin.io/) — Designer and developer collaboration platform. Show designs, assets, and style guides. Free for one project.
   * [Pixelixe](https://pixelixe.com/) — Create and edit engaging, unique graphics and images online.
   * [Responsively App](https://responsively.app) - A free dev tool for faster and more precise responsive web application development.
+  * [SaaS Design](https://www.saasdesign.io/free-figma-templates/) - Free Figma templates to kickstart the UI designs for your next project.
```

Comment:
Seems nice but 6 free designs isnt enough to be called a "service" related to the scope of the list I am afraid, further there is no indication of ownership, rights etc on the site